pom.xml

```diff
@@ -1,12 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You changed quite a few lines, and reordered some elements. This confuses the diff. Could you try to minimise your changes to what is needed, so that the diff is nice and clear?